### PR TITLE
Ming/faq page

### PR DESCRIPTION
**Pull request by mingbeam**
### Problems

- create layout for faq page
- create contact support card
- create english and thai faq pages

- change font in thai faq page
- add table for question 7
- change layout to the old one (with sidebar)
- make sidebar sticky, scrollable in faq pages
- change path to faq pages

- change SideBar, Layout props name
- move link on hover style to Text component(add props)
- link to email when click contact support button

### How would the reviewers verify the solutions

image
image
image

https://user-images.githubusercontent.com/90749495/161331445-7f50ad61-7541-443e-8bfb-100776022a73.mp4

### Preflight checklist

- [ ] **Manual testing** has been performed according to the acceptance criteria.
- [ ] _(heavy style change)_ The solution has been tested on the major browser (Chrome, Firefox, Safari, Microsoft Edge).

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -5,13 +5,16 @@ import Spacer from './Spacer'
...
 import SideBar from './SideBar'
+import ContactSupportCard from './faq_page/ContactSupportCard'
 
 type Props = {
   pageTitle: string
   image: React.ReactNode
   children: React.ReactNode
   contentTitle: string
   contentList: any[]
+  isFaq?: boolean
```
Comment (oukunanan) [inline components/Layout.tsx:16]: This prop is too generic for the FAQ. Please rename what the props are going to do?

File: components/SideBar.tsx [hunk 2]
```diff
@@ -2,10 +2,13 @@ import styles from './SideBar.module.css'
 import Text from './Text'
 import Spacer from './Spacer'
 import { useMediaQuery } from 'react-responsive'
+import classNames from 'classnames'
 
 type Props = {
   contentTitle: string
   contentList: any[]
+  isFaq?: boolean
```
Comment (oukunanan) [inline components/SideBar.tsx:10]: Too generic for FAQ. Please rename what the props are going to do?

File: components/Layout.module.css [hunk 3]
```diff
@@ -17,6 +16,11 @@
   width: 800px;
 }
 
+.content_wrapper a {
```
Comment (oukunanan) [inline components/Layout.module.css:19]: Noted to self: about a tag.

File: components/SideBar.module.css [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+.container.sticky {
+  position: sticky;
+  top: 55px;
+  align-self: flex-start;
+  overflow-y: scroll;
+  height: 100vh;
+  padding-bottom: 50vh;
+}
+
+.container.sticky::-webkit-scrollbar {
```
Comment (oukunanan) [inline components/SideBar.module.css:15]: `-webkit-` is only with  Chrome, Safari, and Opera. How about Firefox and Edge?

File: components/SideBar.module.css [hunk 5]
```diff
@@ -9,6 +22,10 @@
   margin: 0;
 }
 
+.content_list a:hover {
+  text-decoration: underline;
+}
```
Comment (oukunanan) [inline components/SideBar.module.css:27]: Better style this on this [line](https://github.com/Beam-Data-Company/beam-early-access/pull/29/files#diff-2e6066348f1042365f23c2217c2247cd965eb00004f6c871020d618c39b40a99L28)

File: components/faq_page/FaqLayout.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
```
Comment (oukunanan) [inline components/faq_page/FaqLayout.tsx:1]: is this component being use?

File: components/faq_page/ContactSupportCard.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        </Text>
+        <br />
+        <br />
+        <Text color="#ffffff" size={isPhonePortrait ? 14 : 16}>
+          Contact us if you need help with anything. Our support
+          {!isLessThan1160 && <br />} team will get back to you as soon as
+          possible
+        </Text>
+      </div>
+      <button className={styles.button}>Contact Support</button>
```
Comment (oukunanan) [inline components/faq_page/ContactSupportCard.tsx:27]: What happened if click the button?

**Review by oukunanan — CHANGES_REQUESTED** [review]
Please add `https://` of a tag.

https://user-images.githubusercontent.com/86279170/161471501-567c0c6b-f5e9-4c60-a03a-e4b2adee0f8f.mp4

File: components/faq_page/FaqLayout.tsx (hunk 6 above)
Comment (mingbeam) [reply components/faq_page/FaqLayout.tsx:1]: Actually, this file is not being used anymore, so i'm deleting it and its css krub

File: components/faq_page/ContactSupportCard.tsx (hunk 7 above)
Comment (mingbeam) [reply components/faq_page/ContactSupportCard.tsx:27]: will link to mailto:info@beamcheckout.com krub

**Pull request merged**